ush/rocoto/setup_workflow_fcstonly.py

```diff
@@ -688,6 +686,9 @@ def get_workflow(dict_configs, cdump='gdas'):
     if do_vrfy in ['Y', 'YES']:
         dep_dict = {'type':'task', 'name':f'{cdump}vrfy'}
         deps.append(rocoto.add_dependency(dep_dict))
+    if do_metp in ['Y', 'YES']:
```

Comment:
```suggestion
    if cdump in ['gfs'] and do_metp in ['Y', 'YES']:
```
`setup_workflow_expt.py` does take the optional argument of `cdump`.  The `default` is `gfs`, but `gdas` is a valid option, and was working at one point.